middle_end/closure/closure.ml

```diff
@@ -943,7 +948,7 @@ let check_constant_result ulam approx =
           let glb =
             Uprim(P.Pread_symbol id, [], Debuginfo.none)
           in
-          Uprim(P.Pfield i, [glb], Debuginfo.none), approx
+          Uprim(P.Pfield (i, Lambda.layout_any_value), [glb], Debuginfo.none), approx
```

Comment:
I'd like to have a `Lambda.layout_symbol` (which would be weird in lambda...) and use that here (and below)